src/main/java/Task_25/Main.java

```diff
@@ -0,0 +1,103 @@
+package Task_25;
+
+import java.sql.*;
+
+public class Main {
+
+    public static MySQLDriverManager driverManager = MySQLDriverManager.getInstance();
+    public static Connection connection;
+    public static Statement statement;
+    public static ResultSet resultSet;
+
+    static {
+        try {
+            connection = driverManager.getConnection();
+            statement = connection.createStatement();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+
+        createTableOfStudents();
+
+        //Добавление данных в таблицу
+        addStudentToTable(1, "Alex", "Backer");
+        addStudentToTable(2, "Maria", "Crane");
+        addStudentToTable(3, "Ann", "Donovan");
+
+        //Удаление данных из таблицы (по имени студента)
+        deleteStudentFromTable("Alex");
+
+        //Получение данных из таблицы
+        printTableInfo();
+
+        try {
+            if (connection != null) {
```

Comment:
closing of the resource should also be placed in the method, where you initialize and use this resource